src/app/debug/table/table.component.css

```diff
@@ -68,8 +68,11 @@ th {
 }
 .input-field {
   width: 4rem;
+}
+
+.input-group {
```

Comment:
input-group is a bootstrap class that makes it so the input field has that attached box, please give this a different name.

Reply:
changed it to input-group-container